packages/grafana-data/src/vector/FunctionalVector.ts

```diff
@@ -15,32 +15,170 @@ export abstract class FunctionalVector<T = any> implements Vector<T>, Iterable<T
     }
   }
 
+  set(index: number, value: any): void {
+    throw 'unsupported operation';
+  }
+
+  add(value: T): void {
+    throw 'unsupported operation';
+  }
```

Comment:
?

Reply:
Shouldn't FunctionalVector have these implemented as `abstract`? Seems like different vectors either have both set/add or only part of this interface, or none of it which is quite inconsistent. But it feels like standard `array[i]= value`, `array.push(value)`.